libkbfs/folder_branch_ops.go

```diff
@@ -3081,18 +3104,19 @@ func (fbo *folderBranchOps) renameLocked(
 	}
 
 	// does name exist?
-	if de, ok := newPBlock.Children[newName]; ok {
+	replacedDe, ok := newPBlock.Children[newName]
+	if ok {
```

Comment:
Ah, that's because of a future PR (#941) coming later that uses `replacedDe` after that block.  If you feel strongly that we should keep master clean of stuff like this, I'd be happy to fix it here and change it later in that PR, if you want.